ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -334,6 +334,17 @@ StatusSectionLayout {
                             mintPanel.feeText = value
                         }
                     }
+
+                    // TODO: Self-destruct backend
+                    function onSelfDestructFeeUpdated(value) {
+                        // TODO better error handling
+                        if (value === "-") {
```

Comment:
`value` should never be a string or sth like a `-`

Reply:
Same comment as above.. yeah, I agree. It's just an intermediate step. It will be refactored like it is done in `onDeployFeeUpdated` and "-" comparison will be removed!